### PR TITLE
Tomochain and axl usdc

### PR DESCRIPTION
**Pull request by defisaur**
1. https://github.com/trustwallet/assets/pull/26859
2. https://github.com/trustwallet/assets/pull/26915

### CODE REVIEW

**Review by cryptomanz1 — APPROVED** [review]

File: blockchains/mantle/assets/0xEB466342C4d449BC9f53A865D5Cb90586f405215/info.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "name": "Axelar Wrapped USDC",
+    "symbol": "axlUSDC",
+    "type": "MANTLE",
+    "decimals": 18,
```
Comment (vcoolish) [inline blockchains/mantle/assets/0xEB466342C4d449BC9f53A865D5Cb90586f405215/info.json:5]: 6

Comment (defisaur) [reply blockchains/mantle/assets/0xEB466342C4d449BC9f53A865D5Cb90586f405215/info.json:5]: @vcoolish https://github.com/trustwallet/assets/pull/26990

**Review by tawhidnazari57 — COMMENTED** [review]
https://en.webmoney.wiki/projects/webmoney/wiki/E-num

**Comment by Alijocker369** [comment]
> 1. https://github.com/trustwallet/assets/pull/26859
> 2. https://github.com/trustwallet/assets/pull/26915

**Pull request merged**